README.md

```diff
@@ -43,10 +49,10 @@ It should connect to Discord and display the currently logged-in application nam
 
 ## Command reference
 
-1. `!list` - download and list all media files in the text channel this is used in. This needs to be run before `!bust` can be.
-2. `!bust` - Join the vc/stage that the user who ran this command is currently in, and plays the tracks in the channel in order. The user must be in a vc or stage for this to work.
-3. `!skip` - skips the current track :scream: 
-4. `!stop` - stop busting early :scream: :scream: :scream: 
+1. `!list [<channel>]` - download and list all media sent in the current text channel. Specifying a channel will cause songs to be pulled from that channel instead. This must be run before `!bust`.
+1. `!bust` - Join the vc/stage that the user who ran this command is currently in, and plays the tracks in the channel in order. The user must be in a vc or stage for this to work.
+1. `!skip` - skips the current track :scream: 
+1. `!stop` - stop busting early :scream: :scream: :scream: 
```

Comment:
This was an existing issue not related to documenting the new features, but since we're editing these lines anyways we should make capitalization consistent here
```suggestion
1. `!list [<channel>]` - Download and list all media sent in the current text channel. Specifying a channel will cause songs to be pulled from that channel instead. This must be run before `!bust`.
1. `!bust` - Join the vc/stage that the user who ran this command is currently in, and plays the tracks in the channel in order. The user must be in a vc or stage for this to work.
1. `!skip` - Skips the current track :scream: 
1. `!stop` - Stop busting early :scream: :scream: :scream: 
```

Reply:
Looks like this is additionally solved in #42. I'll block this PR until that one merges (and fix any conflicts here).